lib/DDG/Spice/Rain.pm

```diff
@@ -0,0 +1,60 @@
+package DDG::Spice::Rain;
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "is it raining?";
+secondary_example_queries "is it raining in New York City?";
+description "Check if it's raining";
+name "Rain";
+source "Forecast.io";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Rain.pm";
+topics "everyday";
+category "facts";
+attribution web => [ 'https://bibhas.in', 'Bibhas Debnath' ],
+            github => [ 'https://github.com/iambibhas', 'Bibhas'],
+            twitter => ['http://twitter.com/bibhasdn', 'Bibhas D'];
+
+spice to => 'http://forecast.io/ddg?apikey={{ENV{DDG_SPICE_FORECAST_APIKEY}}}&q=$1&callback={{callback}}';
+
+triggers start => "is it raining", "will it rain";
+triggers any => "going to rain";
+triggers end => "raining here", "raining now", "raining today", "raining yet";
```

Comment:
My only concern with these triggers, is that some of them are asking about the weather later (today) and our response still says "it's not raining *now*".

I think we should remove those triggers or adjust the IA to actually answer the given question.